src/rust/ide/json-rpc/src/error.rs

```diff
@@ -28,6 +28,10 @@ pub enum RpcError {
     /// Failed to deserialize message from server.
     #[fail(display = "Failed to deserialize a message: {}.", _0)]
     DeserializationFailed(serde_json::Error),
+
+    /// Response timeout.
+    #[fail(display = "Response timeout after: {:?}.", _0)]
```

Comment:
Can we pretty print to something more like "after {} ms." ? 
When feasible, we want the errors to be pretty, as they might end up being displayed to user.